README.md

```diff
@@ -11,8 +11,8 @@ TBD
 1. [Clone the repo](#1-clone-the-repo)
 2. [Create an IBM Cloud account](#2-create-an-ibm-cloud-account)
 3. [Load data into IBM Db2 on Cloud](#3-load-data-into-ibm-db2-on-cloud)
-4. [Setup Watson studio project](#4-setup-watson-studio)
-5. [Create machine learning model](#5-make-machine-learning-model)
+4. [Setup Watson studio project](#4-setup-watson-studio-project)
+5. [Create machine learning model](#5-create-machine-learning-model)
```

Comment:
Link for step 5 still needs to change:
### 5. Creating and deploying a machine learning model